app.py

```diff
@@ -16,13 +17,16 @@
 
 DB_EMBEDDINGS = "data/embeddings.db"
 DB_SETTINGS = "data/settings.db"
-
+CONVERSATIONS_PATH = 'data/conversations.json'
 
 # Initialize FastAPI app
 app = FastAPI()
 api_app = FastAPI(title="API")
 
-conversations = load_conversations('data/conversations.json')
+if os.path.exists(CONVERSATIONS_PATH):
+    conversations = load_conversations(CONVERSATIONS_PATH)
```

Comment:
what's the point to show empty conversations? i would prefer to fail, or write a detailed error message.

Reply:
Yes, you are true I had removed this empty conversations it is useless. I didn't why I added it But I think I have been wanting to do some other changes.